tests/dualtor/test_orchagent_active_tor_downstream.py

```diff
@@ -85,8 +85,10 @@ def remove_neighbor(ptfhost, duthost, server_ip, ip_version, neighbor_details):
                 yield
         finally:
             ptfhost.shell("supervisorctl start %s" % neighbor_advertise_process)
+            duthost.shell("timeout 0.2 ping -c1 -W1 -i0 -n -q {} || true".format(server_ip))
```

Comment:
Was thinking if we can restart arp_update script and check if neighbor is resolved instead of explicit ping. It will cover the test gap.